presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,81 @@
+* Fix failure when reading a `timestamp` or `timestamp with time zone` value with seconds fraction greater than or equal to 999999999500 picoseconds. ({issue}`6147`)
+* Fix failure when reading a `time` value with seconds fraction greater than or equal to 999999999500 picoseconds. ({issue}`6204`)
+* Fix element representation in arrays returned from `ResultSet.getArray`, making it consistent with `ResultSet.getObject`.
+  Previously the elements were represented using internal client representation (e.g. `String`). ({issue}`6048`)
+* Fix `ResultSetMetaData.getColumnType` for `timestamp with time zone`. Previously the type was miscategorized as `java.sql.Types.TIMESTAMP`. ({issue}`6251`)
+* Fix failure when an instance of `SphericalGeography` geospatial type is returned in the `ResultSet`. ({issue}`6240`)
+* Fix cache entries invalidation, so entries are removed for all users, not only the one that was running the query. ({issue}`6167`)
+
+## CLI Changes
+
+* Fix rendering of `row` values with unnamed fields. Previously they were printed using fake field names like `field0`, `field1`, etc. ({issue}`4587`)
+* Fix query progress reporting. ({issue}`6119`)
+* Fix failure when an instance of `SphericalGeography` geospatial type is returned to the client. ({issue}`6238`)
+
+## Hive Connector Changes
+
+* Reduce scheduling latency for queries where scanned files that are referenced by symlinks
+  are located in the same directory. ({issue}`6158`)
+* Add deserializer class name to split information, which is exposed through the event listener. ({issue}`6006`)
+* Improve parallelism for queries where scanned files are referenced by symlinks. ({issue}`6213`)
+* Add support for S3 streaming uploads. Data is uploaded to S3 as it is written, rather
+  than staged to a local temporary file. This feature is disabled by default, and can be enabled
+  using the `hive.s3.streaming.enabled` configuration property. ({issue}`3712`, {issue}`6201`)
+* Decrease default refresh thread count for metastore cache. ({issue}`6156`)
+* Reduce load on metastore when metastore caching is enabled with background refresh
+  by removing background refresh for partition metadata and statistics. ({issue}`6101`)
+* Verify that data is in the correct bucket file when reading bucketed tables.
+  This is enabled by default, as incorrect bucketing can cause incorrect query results,
+  but can be disabled using the `hive.validate-bucketing` configuration property
+  or the `validate_bucketing` session property. ({issue}`6012`)
+* Allow configuring S3 endpoint in security mapping. ({issue}`3869`)
+* Allow fallback to legacy Hive view translation logic via `hive.legacy-hive-view-translation` config property or 
+  `legacy_hive_view_translation` session property. ({issue}`6195 `)
+    
+## Iceberg Connector Changes
+
+* Optimize predicate pushdown by using filters on non-partition columns during split generation and table scan. ({issue}`4932`)
```

Comment:
Can we express this more in terms of user visible effects? Is a broader set of expressions pushed down?  Can we classify those somehow here?

Reply:
@phd3

Reply:
@losipiuk How about the following? Or may be even just the first sentence of it?

```
Improve performance of queries containing filters on non-partition columns. Such filters are now used for optimizing  split generation and table scan. 
```

Reply:
Looks good to me. Thanks.